### PR TITLE
docs: fix typos and grammar issues

### PR DESCRIPTION
**Pull request by Jeremy-Rivera**
<< Describe the changes >>

Requested changes made

### CODE REVIEW

**Comment by dbjorge** [comment]
I fixed up the remaining comments from my review in #4642 to get this unblocked. The changes look good to me, but @WilcoFiers will need to cover the final review/approval/security approval since I added some commits.

**Review by dbjorge — APPROVED** [review]

**Comment by Jeremy-Rivera** [comment]
Great, once this one gets approved I will perform the nmtoken one

**Review by WilcoFiers — APPROVED** [review]

**Comment by WilcoFiers** [comment]
Thank you for the good work @Jeremy-Rivera!

**Pull request merged**
